tools/buildsteps/windows/vswhere.bat

```diff
@@ -25,26 +25,14 @@ SET vsver=
 
 SET vswhere="%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe"
 
-FOR /f "usebackq tokens=1* delims=" %%i in (`%vswhere% -latest -property installationPath`) do (
+rem Remove -prerelease to NOT find installed prerelease versions of the latest installed version of Visual Studio
+FOR /f "usebackq tokens=1* delims=" %%i in (`%vswhere% -latest -property installationPath -prerelease`) do (
```

Comment:
Is this a temporary thing? Not sure why we would want prerelease to be the default, if anything it should be what it was, with an option to add prerelease